website/server/controllers/api-v3/groups.js

```diff
@@ -861,7 +861,7 @@ api.removeGroupMember = {
       group.memberCount -= 1;
       if (group.hasNotCancelled())  {
         await group.updateGroupPlan(true);
-        await payments.cancelGroupSubscriptionForUser(member, group, true);
+        await payments.cancelGroupSubscriptionForUser(member, group);
```

Comment:
I guess those changes are from the revert of develop? could you merge the latest `develop` changes back ?

Reply:
Do you want me to merge the modal_block branch with develop? I had accidentally merged develop with modal_block branch so I reverted it back before the merge.

Reply:
Yes, pull the latest changes from `develop` into your branch ^^